<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>585</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Equilibrium Calculator__version 0.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">.QWidget{
	background-color:#CCEEFF;
}
.QTextBrowser{
	background-color:#E0FFFF;
	font-size:14px
}
.QComboBox{
	background-color:	#F5F5F5;
	font-size:15px;
}
.QLabel{
	font-size:15px;
	font-weight:490;
	
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Decision Maker 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <item>
           <property name="text">
            <string>アメリカ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>日本</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ドイツ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>イギリス</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>シンガポール</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>スイス</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>スウェーデン</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>フランス</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中国</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>アラブ首長国連邦</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>サウジアラビア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>インド</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ロシア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ブラジル</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>インドネシア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>エジプト</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ルワンダ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ブータン</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ミャンマー</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ガンビア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中央アフリカ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ギニア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ザンビア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ウガンダ</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Decision Maker 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_4">
          <item>
           <property name="text">
            <string>アメリカ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>日本</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ドイツ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>イギリス</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>シンガポール</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>スイス</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>スウェーデン</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>フランス</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中国</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>アラブ首長国連邦</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>サウジアラビア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>インド</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ロシア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ブラジル</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>インドネシア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>エジプト</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ルワンダ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ブータン</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ミャンマー</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ガンビア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中央アフリカ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ギニア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ザンビア</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ウガンダ</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>利得表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser_1">
            <property name="placeholderText">
             <string>利得行列</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>頂点以外のナッシュ均衡</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser_2">
            <property name="placeholderText">
             <string>混合戦略のナッシュ均衡</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>頂点のナッシュ均衡</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser_3">
            <property name="placeholderText">
             <string>頂点のナッシュ均衡</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>動学的均衡状態</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser_4">
            <property name="placeholderText">
             <string>動学的均衡状態</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size:11px;
	font-weight:12;
	font-family:Microsoft YaHei;

}</string>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>Copyright ©2020-2021 KOKO, All Rights Reserved.</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
